README.rst

```diff
@@ -61,10 +63,51 @@ In Scrapy <1.0:
         'scrapy_fake_useragent.middleware.RetryUserAgentMiddleware': 401,
     }
 
+Enabling providers
+---------------------------
+
+Each provider is enabled individually, and used in the order they are defined.
+In case a provider fails execute (it can happen to fake-useragent because of it's dependancy
```

Comment:
Please add "for instance" before "it can happen...". Also link an existing issue for reference.